internal/collector/otelcol.tmpl

```diff
@@ -27,10 +27,35 @@ receivers:
     protocols:
       {{- if eq .Server.Type 0 }}
       grpc:
+        endpoint: "{{ .Server.Host }}:{{ .Server.Port }}"
```

Comment:
Do we want to add/update a unit test for this template update? like: [TestTemplateWrite](https://github.com/nginx/agent/blob/v3/internal/collector/settings_test.go#L51)

Reply:
Oh good suggestion! Implemented